### PR TITLE
Picasso - Release 1 - fix_chart_type_error - Nightly PR

### PR DESCRIPTION
**Pull request by easteregg**
## Issue
Chart was not showing due to Invalid Date was shown instead.

### CODE REVIEW

File: frontend/apps/picasso/components/Organisms/Stats/DailyActiveUsersChart.tsx [hunk 1]
```diff
@@ -39,6 +45,8 @@ export const DailyActiveUsersChart: FC = () => {
     return <ChartLoadingSkeleton />;
   }
 
+  console.log(JSON.stringify(chartSeries, null, 2));
```
Comment (andresmechali) [inline frontend/apps/picasso/components/Organisms/Stats/DailyActiveUsersChart.tsx:48]:
```suggestion
```

File: frontend/apps/picasso/components/Organisms/Stats/DailyActiveUsersChart.tsx [hunk 2]
```diff
@@ -21,7 +26,8 @@ export const DailyActiveUsersChart: FC = () => {
     if (!data) return [];
 
     const tuples: [number, number][] = data.activeUsers.map((activeUser) => {
-      const date = new Date(activeUser.date);
+      console.log(JSON.stringify(activeUser, null, 2));
```
Comment (andresmechali) [inline frontend/apps/picasso/components/Organisms/Stats/DailyActiveUsersChart.tsx:29]:
```suggestion
```

**Review by andresmechali — APPROVED** [review]

**Review by andyjsbell — APPROVED** [review]
Thanks @easteregg

**Comment by easteregg** [comment]
@mergifyio refresh

**Pull request merged**